src/AssetsTransferApi.spec.ts

```diff
@@ -478,4 +555,38 @@ describe('AssetTransferAPI', () => {
 			);
 		});
 	});
+
+	describe('multiLocationAssetIsDestParachainsNativeAsset', () => {
+		it('Should correctly return true when a foreign assets multilocation matches its native chain of origin', () => {
+			const expected = true;
```

Comment:
I think we can just do `expect(isNativeChain).toEqual(true);` instead of saving this to a variable.